lib/matplotlib/rcsetup.py

```diff
@@ -501,6 +501,8 @@ def __call__(self, s):
 
 # a map from key -> value, converter
 defaultParams = {
+    'style': ['', validate_stringlist],
```

Comment:
Actually, getting a list of parent stylesheets is kind of my main idea. This way, you could have really small stylesheets setting only a part of the plot (*e.g.* only the font, or the lines width) that could be associated to create easily bigger stylesheets.